python/grass/pygrass/modules/interface/testsuite/test_modules.py

```diff
@@ -53,6 +53,44 @@ def test_rsun(self):
         out.close()
 
 
+class TestModulesJsonDictExport(TestCase):
```

Comment:
Consider using pytest for these test. No data needed. Assert functionality suited for for plain values.

Reply:
And we can run them faster/in parallel! (I was exploring making pytest run for windows and macOS the other day, but still had problems making it find the pytest I installed on the windows CI)

Reply:
This test case is now removed and replaced with pytest in new file: `pygrass_modules_interface_json_test.py` which has assertations. Hope that is roughly along your lines of thought...